app/routes/how-can-i-help.knowledge.tsx

```diff
@@ -1,26 +1,26 @@
 import {MetaFunction} from '@remix-run/node'
 import DropDown from '~/components/DropDown'
 import HelpItem from '~/components/HowCanIHelp/HelpItem'
-import CategoryCarousel from '~/components/HowCanIHelp/CatgoryCarousel'
+import CategoryCarousel from '~/components/HowCanIHelp/CategoryCarousel'
 import Base from '~/components/HowCanIHelp/Base'
 import VideoImg from '~/assets/video_thumbnail.png'
-import PlayIcon from '~/components/icons-generated/Play'
+import VideoThumbnail from '~/components/VideoThumbnail/videothumbnail'
 
 export const meta: MetaFunction = () => {
   return [{title: 'How Can I Help? - AISafety.info'}]
 }
 
 const NewToAISafety = () => (
   <>
-    <div className="padding-bottom-80">
+    <div className="padding-bottom-56">
       <div className="flexbox padding-bottom-56">
         <h2 className="teal-500 padding-bottom-4 col-8" id="new-to-aisafety">
-          If you're somewhat new to AI safety, we recommend an introductory overview
+          If you{'\u2019'}re somewhat new to AI safety, we recommend an introductory overview
```

Comment:
do we use the `right single quotation mark` consistently for content from gdocs as well as from source code JSX for all the contractions, please? I personally prefer `&apos;`->' for source-code readability, but if ’ comes from gdocs, then `&rsquo;`==`{'\u2019'}`->’ is probably better indeed

Reply:
@LeMurphant would know more but from a quick look at the site it appears there isn't too much consistency:

<img width="756" alt="image" src="https://github.com/user-attachments/assets/4ba663ef-77db-4e3d-8380-0505e1650061">

I personally think the ’ looks prettier than the ', and FWIW that what's the Figma has. I would advocate making that consistent across the site (only) if it's relatively easy